src/main/scala/outwatch/dom/Attributes.scala

```diff
@@ -57,60 +57,60 @@ trait FormEventAttributes {
     *
     * MDN
     */
-  lazy val blur         = new EventEmitterBuilder[InputEvent]("blur")
+  lazy val blur         = EmitterBuilder[InputEvent]("blur")
 
   /** The change event is fired for input, select, and textarea elements
     * when a change to the element's value is committed by the user.
     *
     * MDN
     */
-  lazy val change       = new EventEmitterBuilder[InputEvent]("change")
+  lazy val change       = EmitterBuilder[InputEvent]("change")
 
   /** The focus event is raised when the user sets focus on the given element.
     *
     * MDN
     */
-  lazy val focus        = new EventEmitterBuilder[InputEvent]("focus")
+  lazy val focus        = EmitterBuilder[InputEvent]("focus")
 
   /** The input event is fired when an element gets user input. */
-  lazy val input        = new EventEmitterBuilder[InputEvent]("input")
+  lazy val input        = EmitterBuilder[InputEvent]("input")
 
 
   @deprecated("Deprecated, use 'inputChecked' instead", "0.8.0")
-  lazy val inputBool    = new BoolEventEmitterBuilder("change")
+  lazy val inputBool    = inputChecked
```

Comment:
should we take this release for removing these deprectated methods? The new release will anyhow break some user code.

Reply:
This will be different anyways when using domtypes.